redbot/core/dev_commands.py

```diff
@@ -334,21 +340,21 @@ async def pause(self, ctx, toggle: Optional[bool] = None):
 
     @commands.command()
     @checks.is_owner()
-    async def mock(self, ctx, user: discord.Member, *, command):
```

Comment:
These two commands can be used in DMs, and in DMs the member converter just *picks a random guild* the member is in and returns the member object for that guild. Not sure if `User` would be a better converter, since that will mimic mocked DM behavior more properly and allow mocking of users the bot does not share a server with. Since it's a owner only command, the `fetch_user` call isn't a big deal.

Reply:
Makes sense, I agree with that very much